docs/structured.md

```diff
@@ -0,0 +1,207 @@
+##About##
+
+This document defines the structured data layer that Cockroach exposes on top of
+its distributed key:value datastore.
+
+##Overview##
+
+The structured data layer is an abstraction over the internal key:value store
+that provides familiar data organization tools. The structured data layer
+implements a tabular model of data storage which somewhat resembles a relational
+database at a high level. The terms namespace, table, column, and index/key
+collectively define the structured data abstraction, each roughly akin to
+database, table, column, and index/key in a relational database. The structured
+data layer, when complete, introduces concepts of data storage that many
+database users are familiar with, and greatly increases the range of use-cases
+that Cockroach is able to ergonomically service.
+
+##Goals##
+
+Add support for the following entities: <i>namespaces, tables, columns, and
```

Comment:
instead of using html here, you can do emphasis with an asterisk:
\*emphasis\* -> *emphasis*